nym-vpn-android/nym-vpn-client/src/main/java/net/nymtech/vpn/util/Constants.kt

```diff
@@ -6,7 +6,7 @@ object Constants {
 	const val NYM_VPN_LIB = "nym_vpn_lib"
 
 	// Add Rust environment vars for lib
-	const val DEFAULT_COUNTRY_ISO = "DE"
+	const val DEFAULT_COUNTRY_ISO = "FR"
```

Comment:
I use Switzerland if it does not exist first country from list.

Reply:
I am using FR because I think it is the most consistent country where we have gateways across all of the environments.